runtime/rasdump/dmpagent.c

```diff
@@ -886,7 +886,12 @@ static omr_error_t
 doJavaDump(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
 {
 	J9JavaVM *vm = context->javaVM;
-	if ((0 != strcmp(label, "/STDOUT/")) && (0 != strcmp(label, "/STDERR/"))) {
+
+	if (('-' == label[0]) || (0 == strcasecmp(label, J9RAS_STDOUT_NAME))) {
+			strcpy(label, J9RAS_STDOUT_NAME);
```

Comment:
Please fix the indentation here and two lines below.

Reply:
Sorry, maybe I should stop using Vim to do my code changes....